src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -243,6 +247,21 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Debug,
                 3,
                 "No actions matched the current request. Route values: {RouteValues}");
+
+            _featureNotFound = LoggerMessage.Define<string, string>(
+                LogLevel.Debug,
+                1,
+                "The filter {Filter} could not find the feature {Feature}.");
```

Comment:
There's no real value in parameterizing these things. You also don't need to mention the filter name because that's in the  **source** of the logger.

I'd just say:

> A request body size limit could not be applied. This server does not support the IHttpRequestBodySizeFeature.